R/parse.R

```diff
@@ -169,8 +169,8 @@ parse_connect_rfc3339 <- function(x) {
     # Times with and without offsets require different formats.
     format_string <- ifelse(
       grepl("Z$", .x),
-      "%Y-%m-%dT%H:%M:%SZ",
-      "%Y-%m-%dT%H:%M:%S%z"
+      "%Y-%m-%dT%H:%M:%OSZ",
+      "%Y-%m-%dT%H:%M:%OS%z"
```

Comment:
Was this a bug we hadn't caught here? I see the addition of a test (which is great!) but just curious if this has been wrong and you just caught it here or if something deeper is going on?

Reply:
Neither — `%S` was valid for most of the timestamps Connect sends, but logs contain decimal second precision, so to properly parse the log data from the `v1` endpoint, we need `%OS`. The existing tests confirm that this change preserves compatibility with timestamps lacking decimal seconds.